suricata.yaml.in

```diff
@@ -1953,7 +1953,13 @@ pfring:
     cluster-id: 99
 
     # Default PF_RING cluster type. PF_RING can load balance per flow.
-    # Possible values are cluster_flow or cluster_round_robin.
+    # Possible values are:
+    # - cluster_flow:               6-tuple: <src ip, src_port, dst ip, dst port, proto, vlan>
+    # - cluster_inner_flow:         6-tuple: <src ip, src port, dst ip, dst port, proto, vlan>
```

Comment:
Naive question: Since both use the same 6 tuple, what's the difference between the two?

Reply:
That's a good question --- this is how the PF_RING kernel include header refers to them:
https://github.com/ntop/PF_RING/blob/dev/kernel/linux/pf_ring.h#L1003:L1020